### ISSUE TITLE
L&F breaks tab navigation

### ISSUE THREAD

**Issue by cowwoc** [Revivius/nb-darcula#114]
Following up on https://github.com/Revivius/nb-darcula/issues/101, keyboard navigation is still broken.
- In Netbeans 8.1 final, keyboard navigation works regardless of whether Darcula is installed.
- In dev builds, keyboard navigation works with a vanilla install but fails the minute Darcula 1.5 is installed.

This is reproducible in:

```
Product Version: NetBeans IDE Dev (Build 201606270002)
Java: 1.8.0_92; Java HotSpot(TM) 64-Bit Server VM 25.92-b14
Runtime: Java(TM) SE Runtime Environment 1.8.0_92-b14
System: Windows 7 version 6.1 running on amd64; UTF-8; en_US (nb)
```

**Comment by cowwoc** [comment]
I just double-checked and this bug is reproducible in Netbeans 8.2 (which was just released).

**Comment by RomkeVdMeulen** [comment]
Same here.

**Comment by AlexFalappa** [comment]
Seems that Ctrl+PageUp/Down for editor tab switching breaks often!

It's currently broken in NetBeans 8.2 but only with Nimbus and, unfortunately, Darcula LAFs due to the newly introduced pinnable watches feature.

See, and please vote, https://netbeans.org/bugzilla/show_bug.cgi?id=267807

**Comment by markiewb** [comment]
@AlexFalappa: Can you please confirm that the PR #128 fixes this issue?!

**Comment by AlexFalappa** [comment]
@markiewb I have built and tested your PR source and indeed it seems the problem is fixed.

Now it would be great to port your fix over to Nimbus LAF as well. As @cowwoc also mentioned it would be great to expose the input map setting method at NetBeans api level.

**Comment by AlexFalappa** [comment]
The related NetBeans bug has been fixed. Nimbus LAF should work too (not tested myself).

Should be in a future patch or minor version.

**Comment by josenicomaia** [comment]
I am still with this bug :(
